main.cpp

```diff
@@ -91,7 +91,92 @@ good to go!
  Wait for my code review.
  */
 
+
+
+
+
 #include <iostream>
+
+struct FloatType
+{
+    float add(float lhs, float rhs )
+    {
+        return (lhs + rhs);
+    }
+    float subtract( float lhs, float rhs )
+    {
+        return (lhs - rhs);
+    }
+    float multiply( float lhs, float rhs )
+    {
+        return (lhs * rhs);
+    }
+    float divide( float lhs, float rhs )
+    {
+        if (rhs == 0)
```

Comment:
you're comparing a `float` (`rhs`) with an `integer` (`0`).  use correct types when comparing to suppress the warnings that have appeared.
- [ ] revise